app/src/angular/configs/app.run.js

```diff
@@ -85,7 +85,8 @@ function AppRun($rootScope, $log, $window, $timeout, $interval, $q, $state, $mdD
             escapeToClose: false,
             locals: {
                 args: {
-                    symbol: symbol
+                    symbol: symbol,
+                    isERC20TokensAllowed : false
```

Comment:
instead of hardcode set *false* add arg to method.. name it **allowSelectERC20Token**